OpenImis.ePayment/Escape/Payment/Data/ImisPayment.cs

```diff
@@ -451,7 +452,47 @@ Amount DECIMAL(18, 2)
 
         }
 
-        
+        public int ControlNumbersToBeRequested(string productCode)
+        {
+            var sSQL = @";WITH TotalProductUsage
+                        AS
+                        (
+	                        SELECT CASE WHEN COUNT(1)/3 < 200 THEN 200 ELSE COUNT(1)/3 END  Last3MonthsEnrollment 
```

Comment:
Yes, if we find a product which is not used in last 3 months or hardly used then request 200 CNs. We give 100 CNs when fetched by EO. We keep 100 extra in case another EO wants CNs for the same product. Because the consumption is no to too low we keep 200 just to accommodate one more EO